opba-protocols/protocol-bpmn-shared/src/test/java/de/adorsys/opba/protocol/bpmnshared/config/flowable/JsonCustomSerializerTest.java

```diff
@@ -0,0 +1,51 @@
+package de.adorsys.opba.protocol.bpmnshared.config.flowable;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+import java.util.Random;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class JsonCustomSerializerTest {
+    private static JsonCustomSerializer serializer;
+
+    @BeforeAll
+    public static void init() {
+        serializer = new JsonCustomSerializer(null, new ObjectMapper(), Arrays.asList("java.lang.String"), 2048);
+    }
+
+    @Test
+    void serializeLessThanMaxValueTest() {
+        String input = generateRandomStringOfLength(2045);
+        assertThat(serializer.isAbleToStore(input)).isTrue();
+    }
+
+    @Test
+    void serializeEqualsToMaxValueTest() {
+        String input = generateRandomStringOfLength(2046);
+        assertThat(serializer.isAbleToStore(input)).isTrue();
+    }
+
+    @Test
+    void serializeMoreThanMaxValueTest() {
+        String input = generateRandomStringOfLength(2047);
+        assertThat(serializer.isAbleToStore(input)).isFalse();
+    }
+
+    private String generateRandomStringOfLength(int length) {
```

Comment:
Instead of this long function - 
```
Strings.repeat
```
from Guava would solve the issue

Reply:
Replaced with Strings.repeat in all tests